model_db.py

```diff
@@ -37,3 +39,26 @@ def init_from_disk(self, filepath: str = "models"):
             model = model_name.split("--")[-1]
             self[model] = SentenceTransformer(model_path)
             log.info(f"Loaded model {model}")
+
+    def embed_sentences(self, model: str, sentences: list[str]):
+        model_obj = self[model]
```

Comment:
Do we need a check here for that model existing? And committing to logs if it doesn't

Reply:
yes we should. ill raise a ticket